.vscode/settings.json

```diff
@@ -1,4 +1,12 @@
 {
+    "[mdx]": {
+        "editor.wordWrapColumn": 120,
```

Comment:
Caps line-length during editing of MDX and Markdown files to prevent endless horizontal scroll